tests/services/pids/providers/test_oai_invenio_pid_provider.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
I feel that we also need a test that checks the full E2E service flow works (i.e. publishing a record via services, and an OAI PID gets created).

Reply:
added a testcase publishing a record and checking existence of OAI PID in `tests/services/test_rdm_service.py`